tests/unit/components/SharedComponents/PhotoCount/PhotoCount.test.js

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react-native";
+import { PhotoCount } from "components/SharedComponents";
+import React from "react";
+
+describe( "PhotosCount", () => {
+  it( "should render correctly", () => {
+    render( <PhotoCount count={10} /> );
+
+    const photoCount = screen.queryByTestId( "photo-count" );
+
+    expect( photoCount ).toBeTruthy();
```

Comment:
Here it would be nicer, to assert for the correct text to be shown, in this case, 10.